dsl/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/expression/CurrentDateDslTest.kt

```diff
@@ -0,0 +1,25 @@
+package com.linecorp.kotlinjdsl.dsl.jpql.expression
+
+import com.linecorp.kotlinjdsl.dsl.jpql.queryPart
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expression
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.Expressions
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+import java.sql.Date
+
+class CurrentDateDslTest : WithAssertions {
+    @Test
+    fun `currentDate()`() {
```

Comment:
If the name of the test exposes only the name of the method being tested, it is not parenthesized. Please modify it like this:

```kotlin
@Test
fun currentDate() {
  // ...test codes
}
```